app/Console/ConsoleServiceProvider.php

```diff
@@ -76,6 +77,7 @@ final class ConsoleServiceProvider extends AbstractServiceProvider
         'console.organizer:change-owner-bulk',
         'console.label:bulk-exclude',
         'console.label:exclude',
+        'console.label:exclude-malformed',
```

Comment:
Maybe a better wording is `exclude-invalid` because they no longer match the required validation rules.